src/main/java/seedu/duke/assets/DoctorList.java

```diff
@@ -2,18 +2,50 @@
 
 import java.util.ArrayList;
 
+
 public class DoctorList {
 
-    ArrayList<Doctor> doctorList;
+    private boolean isNull(String string) {
+        return string == null;
+    }
+
+    private ArrayList<Doctor> doctors = new ArrayList<>();
+
+    public Doctor getDoctor(String nric) {
+        for (Doctor doctor : doctors) {
+            if (doctor.getNric().equals(nric)) {
+                return doctor;
+            }
+        }
+        return null;
+    }
 
 
     //view particular doctor
     public void viewDoctor(String nric) {
-
+        if (isNull(nric)) {
+            return;
+        }
+        System.out.println(getDoctor(nric).toString());
     }
 
     //view all doctor
     public void viewDoctor() {
+        System.out.println(doctors.toString());
+    }
+
+    //get the number of doctors
+    public int getSizeDoctor() {
+        return doctors.size();
+    }
+
+    //remove the specific doctor
+    public void removeDoctor(int index) {
+        doctors.remove(index);
+    }
 
+    @Override
+    public String toString() {
+        return "aaa";
```

Comment:
Should this placeholder be changed soon?

Reply:
yup i was wondering if we should change in this version or the next one

Reply:
i think its this version, cuz the app needs the view all doctors and view a particular doctor fn

Reply:
made changes to the placeholder pls help check thanks

Reply:
hi anvitha, i believe you are missing the toString method under the doctor class!